emcc.py

```diff
@@ -181,6 +181,10 @@ def run():
     print shared.get_llvm_target()
     exit(0)
 
+  elif '--cflags' in sys.argv:
+    print ' '.join(shared.COMPILER_OPTS + ['-emit-llvm', '-c'])
```

Comment:
Does this line duplicate '-emit-llvm' and '-c'? That seems risky. Actually come to think of it '-c' and 'emit-llvm' probably don't belong in a list of cflags anyway, since they affect the type of output rather than the language or compiler settings. For example if you wanted to use those same flags with upstream wasm clang, you wouldn't use -emit-llvm, and if you wanted to build a binary instead of an object file you wouldn't want -c.